snap/snaptest/snaptest.go

```diff
@@ -55,6 +55,23 @@ func MockSnap(c *check.C, yamlText string, sideInfo *snap.SideInfo) *snap.Info {
 	return snapInfo
 }
 
+// MockInfo parses snap.yaml and returns a validated snap.Info object.
```

Comment:
... parses the given snap.yaml text and returns ... object including the optional sideInfo.